migrations/m151218_234654_add_timezone_to_profile.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Dektrium project.
+ *
+ * (c) Dektrium project <http://github.com/dektrium/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use yii\db\Schema;
+use yii\db\Migration;
+
+class m151218_234654_add_timezone_to_profile extends Migration
+{
+    public function up()
+    {
+        $this->addColumn('{{%profile}}', 'timezone', Schema::TYPE_STRING . (255));
```

Comment:
Moreover, do we really need 255 chars here? What is the longest possible timezone name?